build.js

```diff
@@ -728,57 +661,108 @@ function copyResources() {
         'sourceMappingURL=core.js.map'
     );
 
-    fs.writeFile(corejsPath, result, 'utf8', function(err) {
+    fs.writeFileSync(corejsPath, result, 'utf8', (err) => {
       if (err) return console.log(err);
     });
   });
-}
+};
 
-async function build(bcd, webref) {
+const buildManifest = (tests) => {
```

Comment:
Not reviewing this in detail, but I note that it's not a very long function, so the server should be able to compute all of this information from the tests listing at runtime. (See also comment in app.js about `/tests/*` for how I think this can change.)